app/importers/concerns/importable.rb

```diff
@@ -1,10 +1,16 @@
-module Importer
+module Importable
+  extend ActiveSupport::Concern
   # The module provides functionality useful for importing source data, and
   # can be included into any class that will do so.
 
-  def self.included(base)
-    base.class_eval do
-      base.send(:prepend, ParentInstanceMethods)
+  included do
+    send(:prepend, ParentInstanceMethods)
+
+    class << self
+      attr_accessor :disabled
+      def disabled?
+        !!disabled
```

Comment:
You mean you don't like the "bang bang you're boolean" operator?!? :smile: 

In all practical use cases `disabled` will be either `true` or `nil` (it might get set to `false`, but other than that nothing else I'd hope). I could add more code that defaults `disabled` to false, and also validates its value, but I don't see any point in doing that. If you put `self.disabled = 'donkey'` in your importer then you should expect problems :smiley:

`present?` is a Rails thing, and is aimed at strings that might be empty or contain only whitespace. Since I'm dealing with only `nil` and `true`, it seems a bit too sophisticated for our needs. The double-bang is there to just convert `nil` to `false`, so that `disabled?` returns a boolean. And it's pure Ruby code, which probably makes it faster although I didn't break out the profiler so I don't know that for sure :wink: